app/assets/javascripts/compute_resources/ovirt/nic_info.js

```diff
@@ -1,3 +1,3 @@
 providerSpecificNICInfo = function(form) {
-  return form.find('.ovirt_name').val() + ' @ ' + form.find('.ovirt_network').val();
+  return form.find('.ovirt_name').val() + ' @ ' + form.find('.ovirt_network').text();
```

Comment:
Do you know what exactly that fixes?  Is it a bug with something like select2?  (If so, please file it separately as we'd take it for 1.9.)